frontend/src/components/ApplicationHeader.tsx

```diff
@@ -0,0 +1,120 @@
+import { Button } from "@tritonse/tse-constellation";
+import { ChevronDown } from "lucide-react";
+import { useEffect, useState } from "react";
+import { twMerge } from "tailwind-merge";
+
+import api, { Application } from "../api";
+import { formatPhoneNumber } from "../helpers/application";
+import { formatFieldNameHumanReadable } from "../helpers/review";
+import { useAlerts } from "../hooks/alerts";
+import { formatQuarter } from "../util";
+
+function PromptDropdown({ title, content }: { title: string; content: string | undefined }) {
+  const [isOpen, setIsOpen] = useState(true);
+
+  return (
+    <div className="tw:rounded-lg tw:overflow-hidden">
+      <button
```

Comment:
For the prompt dropdowns and score cards, can we apply the black border to the header as well as the body? So a black border around the entire thing, plus a black line between header/body. I think it would look better that way; the header without a border looks kinda weird imo

Reply:
<img width="413" height="347" alt="Screenshot from 2025-09-14 20-07-46" src="https://github.com/user-attachments/assets/93bc9b1c-dda2-427e-abb6-117edc7b162b" />
<img width="1188" height="368" alt="Screenshot from 2025-09-14 20-07-37" src="https://github.com/user-attachments/assets/cfbfbcd4-1c9a-4f46-a09a-9302d5b535ae" />